package-lock.json

```diff
@@ -4136,20 +4603,17 @@
         "code-point-at": {
           "version": "1.1.0",
           "bundled": true,
-          "dev": true,
-          "optional": true
+          "dev": true
         },
         "concat-map": {
           "version": "0.0.1",
           "bundled": true,
-          "dev": true,
-          "optional": true
+          "dev": true
         },
         "console-control-strings": {
           "version": "1.1.0",
           "bundled": true,
-          "dev": true,
-          "optional": true
```

Comment:
I can go through and undo these changes if they're not desired.

Reply:
My impression right now is that this is just the new way of things in npm world.

Reply:
👍